Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
```

Comment:
Ok, I understand but it is really an open-source project as you can see at [github/red](https://github.com/red/red).
From the red binary you can compile (and even cross-compile) any red source to binary.